lang/lib/charset.py

```diff
@@ -14,15 +14,117 @@
 # This transformation is applied to the translation prior to being converted to the final encoding,
 # and maps UTF8 to UTF8. It replaces unavailable symbols in the translation to a close
 # representation in the source encoding.
+# sources
+# https://en.wikipedia.org/wiki/Czech_orthography
+# https://en.wikipedia.org/wiki/German_orthography
+# https://en.wikipedia.org/wiki/French_orthography
+# https://en.wikipedia.org/wiki/Spanish_orthography
+# https://en.wikipedia.org/wiki/Italian_orthography
+# https://en.wikipedia.org/wiki/Polish_alphabet
+# https://en.wikipedia.org/wiki/Dutch_orthography
+# https://en.wikipedia.org/wiki/Romanian_alphabet
+# https://en.wikipedia.org/wiki/Hungarian_alphabet
+# https://en.wikipedia.org/wiki/Gaj%27s_Latin_alphabet
+# https://en.wikipedia.org/wiki/Slovak_orthography
+# https://en.wikipedia.org/wiki/Swedish_alphabet
+# https://en.wikipedia.org/wiki/Norwegian_orthography
+
 TRANS_CHARS = {
-    'Ä': 'ä',
-    'Å': 'A',
-    'Ö': 'ö',
-    'Ü': 'ü',
-    'å': 'a',
-    'æ': 'ä',
-    'ø': 'ö',
-    'ß': 'ss',
+    'á': 'a', #cz,fr,es,hu,sk
+    'Á': 'A', #cz,fr,hu,sk
+    'à': 'a', #fr,it
+    'À': 'A', #fr,it
+    'â': 'a', #fr,ro
+    'Â': 'A', #ro
+    'Ä': 'ä', #de,sv,no,sk
```

Comment:
should be 'Ä': 'A', #de,sv,no,sk

and add

should be 'ä': 'a', #de,sv,no,sk

Reply:
It is like that so that it uses the builtin ä character. Unfortunately, there is no uppercase Ä character in the LCD ROM.

Reply:
@ingbrzy Is that solved?

Reply:
> @ingbrzy Is that solved?
yes solved